### ISSUE TITLE
MODL | D-FINE | Trening v1

### ISSUE THREAD

**Issue by Johannestj** [kartverket/BirdAI#135]
### Oppgavebeskrivelse
**Arbeidsoppgave:** Treng versjon 1 av D-FINE

**Kontekst:** VI har slitt med yolov9, så vi tar da å trener D-FINE.

### Definition of Done (DoD)
**Krav for fullført oppgave:** Modellen nedlastet i en fil, lagt i relevant kanal på discord.

### Ved oppretting av Artikkel/Item

| **Situasjon**       | **Selve Prosedyren**           |
|-------------|-----------------------| 
| Artikkel/Item == New | Fyll ut alle seksjoner av malen (Arbeidsoppgave, Kontekst, DoD (Definition of Done)) |
| Start- og Enddate | Ved start på en oppgave, legg til dato, og tilsvarende ved slutt.       | 

### Ved flytting av Artikkel/Item
| **Situasjon**       | **Selve Prosedyren**           |
|-------------|-----------------------|
| Ved item fra New → Todo | Alle merkander skal være fylt, samt tittel og beskrivelse.       | 
| Ved item fra Todo → In Progress | Legg på Start date (== dagens dato) |
| Ved item fra In Progress → In Review | Legg til kommentar til "revisor", tagg "revisor" og legg "revisor" til som Assigne |
| Ved item fra In Review → Done! | Legg til End date (== dagens dato). Kommenter på "revisors" tilbakemelding, og evt. kommentarer til videre ved label: Revisiting|

**Comment by martinstereo** [comment]
Møter på problem med D-FINE. 

![Image](https://github.com/user-attachments/assets/c68c0f32-980c-4a7c-87ed-4b9afe50e182)

**Comment by martinstereo** [comment]
Trente modell nå: 
| Responsible | Model name | Pre-trained | Size (params) | Dataset | Img size (px) | mAP  | Area=small | Recall | Area=small | Latency (TensorRT: T4, FP16, bs=1) | Epochs | GPU  |
|------------|-----------|-------------|--------------|---------|--------------|------|------------|--------|------------|----------------------------------|--------|-----|
| Martin    | D-FINE    | COCO        | s (10)       | Versjon 2 NY | 640  | 0.57 | 0.169      | 0.469  | 0.469      | N/A                              | 77     | A100 |
